components/Wizard/CreateImageWizard.js

```diff
@@ -151,6 +152,18 @@ const CreateImageWizard = (props) => {
     if (blueprint.customizations) {
       formState["customizations-hostname"] = blueprint.customizations.hostname;
       formState["customizations-install-device"] = blueprint.customizations.installation_device;
+      formState["customizations-users"] = [];
+      if (blueprint.customizations.user?.length) {
+        blueprint.customizations.user.forEach((user) => {
+          const formUser = {
+            name: user.name,
+            password: user.password,
+            "is-admin": user.groups?.includes("wheel"),
+            "ssh-key": user.key,
```

Comment:
Not really a problem but for my own knowledge... I suppose `"is-admin"` and `"ssh-key"` are strings because otherwise you can't have a `-` in a property name, but why not just use camelCase? I think with the kebab-case we won't be able to access these properties using dot notation, right?

Reply:
Done to be consistent with Data Driven Forms.